### ISSUE TITLE
Missing particles on the syringe gun after 2.1.2 patch shipped.

### ISSUE THREAD

**Issue by AxelFreeze14** [tf2classic/tf2classic-issue-tracker#271]
After the new 2.1.2 patch arrived, I've been getting a bunch of missing particle effects every time the medic shoots the syringe gun. (I tried reinstalling the game 3 times, and it still shows up).
![Missing particles (after shooting)](https://github.com/tf2classic/tf2classic-issue-tracker/assets/145644288/2cc893be-49ed-4f76-b16b-6d8eb4a16b36)
![Missing particles (before shooting)](https://github.com/tf2classic/tf2classic-issue-tracker/assets/145644288/5596a84e-33e9-420f-8dec-cac41f2b24b7)
![Missing particles (console errors)](https://github.com/tf2classic/tf2classic-issue-tracker/assets/145644288/50d6aa2e-ec2b-4dce-860a-1598fac91b85)
![Missing particles (3rd person view)](https://github.com/tf2classic/tf2classic-issue-tracker/assets/145644288/a83cb5f6-01eb-4fc8-a566-2eae849b401a)

**Comment by sapphonie** [comment]
type `flush_dls` in console, delete all the `.cache` files you can find, and reinstall your game fully

**Comment by sapphonie** [comment]
also post specs, enable crash reporting (`cl_send_error_reports 1`), restart your game, post your steamid, and type `sentry_test` in console, then report back

**Comment by AxelFreeze14** [comment]
I did everything and i still got missing particles. Also here are my specs and SteamID.
Laptop ASUS VivoBook 17 Model X712JA
OS: Windows 10 Home x64
CPU: Intel Core i7-1065G7 @ 1.30GHz/1.50GHz
RAM: 16.0 GB
![Still missing particles](https://github.com/tf2classic/tf2classic-issue-tracker/assets/145644288/ad1176e8-a429-4ce0-97f4-7ba3344a8065)
![SteamID](https://github.com/tf2classic/tf2classic-issue-tracker/assets/145644288/b3133317-43a7-4be8-ae50-ec7b4adf6fcd)

**Comment by AxelFreeze14** [comment]
Nevermind, i finally got it fixed, thanks for the flush_dls command.

**Comment by sapphonie** [comment]
You're welcome, glad to help

**Comment by AxelFreeze14** [comment]
I think i figured out what was the issue, it was the "rate" command. When it was below the default 80,000 value, the syringe gun particles get missing.

**Comment by HelpMePleasepls** [comment]
I'm also encountering the same issue. I'm on OpenSUSE Tumbleweed.
![Screenshot_20240915_222653](https://github.com/user-attachments/assets/ff1bbda5-d1e5-4145-ac1e-a99b1ef4d925)
I've tried sentry_test, and maybe I didn't wait long enough, but nothing happened and it did not resolve the issue.
I've tried flush_dls and to delete the contents of .cache, still did not resolve the issue.
I have crash reporting on. My SteamID is "jonohe" (this is what it looks like in the link: steamcommunity.com/id/jonohe/)
